src/routes/solid-start/advanced/websocket.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Websocket Endpoint"
+---
+
+Websocket endpoint may be included by passing the ws handler file you specify in your start config. Note that this feature is experimental on the Nitro server and its config may change in future releases of Solid Start. Use it with caution.
+
+```js
+import { defineConfig } from "@solidjs/start/config";
+
+export default defineConfig({
+	server: {
+		experimental: {
+			websocket: true,
+		},
+	},
+}).addRouter({
+	name: "party",
+	type: "http",
+	handler: "./src/ws.ts",
+	target: "server",
+	base: "/ws",
+});
+```
+
+Inside the ws file, you can export a partyHandler function to manage WebSocket connections and events:
+
+```tsx
+import { partyHandler } from "vinxi/party";
+
+export default partyHandler({
+	onConnect(party, conn) {
+		conn.send("Hello from the server!: " + conn.id);
```

Comment:
you used a template string below, would you mind making this snippet consistent using a template string here as well?